ci-operator/config/openshift/compliance-operator/openshift-compliance-operator-master.yaml

```diff
@@ -7,18 +7,22 @@ base_images:
     name: builder
     namespace: ocp
     tag: golang-1.14
+  ocp_builder_golang-1.15:
```

Comment:
Why not add another builder and not drop the 1.14 builder above? It doesn't seem to be used anywhere anymore...